src/sysusers/sysusers.c

```diff
@@ -19,7 +19,9 @@
 
 #include <getopt.h>
 #include <grp.h>
+#ifdef ENABLE_GSHADOW
 #include <gshadow.h>
+#endif
```

Comment:
why this? gshadow appears to be unconditional part of the glibc API even if not used locally...